modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalFolderServiceImpl.java

```diff
@@ -130,6 +130,21 @@ public JournalFolder getFolder(long folderId) throws PortalException {
 		return folder;
 	}
 
+	@Override
+	public JournalFolder getFolderByExternalReferenceCode(
+			long folderId, String externalReferenceCode)
```

Comment:
I guess you meant `groupId` instead of `folderId`

Reply:
You are right Carlos!! I will change it